### ISSUE TITLE
Percentage is not

### ISSUE THREAD

**Issue by jahewson** [IndigoUnited/node-request-progress#24]
So I was trying to work out what was wrong with my code until I saw this:

``` javascript
percentage: 0.5,            // Overall percentage (between 0 to 1)
```

Percentages are between 0 and 100. That's what makes them percentages 😄 , otherwise it's just a `ratio`.

There's a non-breaking way to fix this, which is to keep the existing behaviour as undocumented and to add a new `percent` field with the correct behaviour.

**Comment by satazor** [comment]
You are right, what if we name it progress or completed or something similar? I don't really like ratio, too generic.

**Comment by satazor** [comment]
I'm also ok to add percent and removing percentage (major semver bump)

**Comment by satazor** [comment]
@jahewson Changes released as v3.0.0.

**Comment by Ltrlg** [comment]
I do not understand why this issue is closed since you just renamed the field without fixing the wrong behavior (`percent` still has values [between 0 and 1](https://github.com/IndigoUnited/node-request-progress/blob/master/index.js#L75) instead of between 0 and 100).
